src/index.html

```diff
@@ -8,6 +8,71 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--0">
+      <span class="stars__star"></span>
+      
+      <span class="stars__star"></span>
+      
+      <span class="stars__star"></span>
+      
+      <span class="stars__star"></span>
+      
+      <span class="stars__star"></span>
+    </div>
+    <div class="stars stars--1">
+      <span class="stars__star"></span>
+      
+      <span class="stars__star"></span>
+      
+      <span class="stars__star"></span>
+      
+      <span class="stars__star"></span>
+      
+      <span class="stars__star"></span>
+    </div>
+    <div class="stars stars--2">
+      <span class="stars__star"></span>
+      
+      <span class="stars__star"></span>
+      
+      <span class="stars__star"></span>
+      
+      <span class="stars__star"></span>
+      
+      <span class="stars__star"></span>
+    </div>
```

Comment:
blank lines needed between _multiline_ blocks

```suggestion
    <div class="stars stars--0">
      <span class="stars__star"></span>
      <span class="stars__star"></span>
      <span class="stars__star"></span>
      <span class="stars__star"></span>
      <span class="stars__star"></span>
    </div>
    
    <div class="stars stars--1">
      <span class="stars__star"></span>
      <span class="stars__star"></span>
      <span class="stars__star"></span>
      <span class="stars__star"></span>
      <span class="stars__star"></span>
    </div>
    
    <div class="stars stars--2">
      <span class="stars__star"></span>
      <span class="stars__star"></span>
      <span class="stars__star"></span>
      <span class="stars__star"></span>
      <span class="stars__star"></span>
    </div>
```